realm/realm-library/src/objectServer/java/io/realm/mongodb/App.java

```diff
@@ -330,6 +330,20 @@ public User switchUser(User user) {
         return user;
     }
 
+    /**
+     * Removes a users credentials from this device. If the user was currently logged in, they
+     * will be logged out as part of the process. This is only a local change and does not
+     * affect the user state on the server.
+     *
+     * @param user to remove
+     * @return user that was removed.
+     * @throws AppException if called from the UI thread or if the user was logged in, but
+     *                      could not be logged out.
+     */
+    public User removeUser(User user) throws AppException {
```

Comment:
We also need to move the `removeUserAsync()` method.

Reply:
Didn't see the method 🤦🏻‍♂️